FrameworkInternals/parser_module.py

```diff
@@ -99,9 +99,13 @@ def formatter_class(prog): return QuasarArgsHelpFormatter(
     # create the parser for the "generate config_doc" command
     generate_config_doc_subparser = generate_subparsers.add_parser(
         'config_doc', help='Generates documentation of the Configuration Schema. Such a documentation is a HTML document with description of all fields that might appear in the configuration')
+    generate_config_doc_subparser = generate_subparsers.add_parser(
+        'config_doc_md', help='Generates documentation of the Configuration Schema. Such a documentation is a Markdown document with description of all fields that might appear in the configuration')
     # create the parser for the "generate as_doc" command
     generate_as_doc_subparser = generate_subparsers.add_parser(
-        'as_doc', help='Generates address space documentation (the documentation of all OPCUA ObjectTypes, Variables, etc… ) and places it in Documentation/AddressSpaceDoc.html')
+        'as_doc', help='Generates address space documentation in a html document (the documentation of all OPCUA ObjectTypes, Variables, etc… ) and places it in Documentation/AddressSpaceDoc.html')
```

Comment:
Change ‘html’ to ‘HTML’ for consistency.

Reply:
Done